tests/Browser/2-critical.testcafe.js

```diff
@@ -42,7 +42,7 @@ test("Applicant Profile - My Skills", async t => {
     .expect(Selector("h3").withText("Writing my application").visible)
     .ok()
     .click(Selector("button").withText("Got it!"))
-    .wait(300)
+    .wait(1234)
```

Comment:
I think this is covered by #2516

Reply:
Ya so I fixed the broken test here first. Then noticed it in #2499 PR as well. Thats why I made a PR to fix it.